Classes/Private/Manager/Objects/OCTCall.m

```diff
@@ -55,41 +35,4 @@ - (NSUInteger)hash
     return [self.chat.uniqueIdentifier hash];
 }
```

Comment:
In this case I am overriding it since I want the OCTCalls to be compared by what OCTChats they belong to. Otherwise I would have to do something like:
`XCTAssertEqualObjects(call1.chat, call2.chat)`
rather than
`XCTAssertEqualObjects(call1, call2)`

Let me know if I'm overlooking something.